Highrachy/ballers-ws
# #173327352 Update favicon

#### What does this PR do?
This PR updates favicon for Ballers landing page. The previous favicon doesn't work well when bookmarked in some phones, hence, the design team updated the favicon to fix this.

#### Description of Task to be completed?
The following favicons are created to ensure that all devices are catered for:

* android-chrome-192x192.png
* android-chrome-512x512.png
* apple-touch-icon.png
* favicon-16x16.png
* favicon-32x32.png
* favicon.ico
* site.manifest

#### How should this be manually tested?
Visit Ballers landing page
Refresh your browser multiple times (till you see the favicon as shown below)
